scripts/generate_release_notes.py

```
@@ -133,32 +134,30 @@ def custom_order(pr):
     def add_pull_request(self, pull_request):
         """Add pull request information to the release notes dict."""
         author = pull_request.user.login if pull_request.user else None
+        pr_number = pull_request.number
         if author in IGNORED_CONTRIBUTORS:
-            print(f"Skipping #{pull_request.number} {pull_request.title} by {author}")
+            print(f"Skipping #{pr_number} {pull_request.title} by {author}")
+            self.skipped_pull_requests.add(pr_number)
             return None
```

Ruff (0.15.9)

[warning] Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)